src/libertem/executor/dask.py

```diff
@@ -151,7 +160,8 @@ def _make_items_all():
         if all_nodes:
             items = _make_items_all()
         else:
-            items = ((lambda: fn(p), p.get_locations())
+            # TODO check if we should request a compute resource
```

Comment:
... or possibly a "service" resource? Depends on the `fn`, I guess - current usage is only "service".

Reply:
Currently, service doesn't request any resource, that means it can run on any free worker. Would it make sense to restrict service to only specific workers?

Reply:
Right. I don't know if we should restrict - I think we can see if there are any problems with the unrestricted case. I think currently worst case a GPU worker will be doing non-GPU work?

Reply:
Yes, that is the worst that can happen. Let's see if this becomes an issue in the future.